website/docs/docs-section/faq.md

```diff
@@ -4,15 +4,65 @@ title: FAQ
 sidebar_label: FAQ
 ---
 
-#### What are OpenAttestation documents?
+### What are OpenAttestation documents?
 
 OpenAttestation documents (OA documents) are a type of verifiable documents or verifiable credentials.
 
 As our transactions become increasingly digital, there is a need to ensure that the legitimacy of credentials such as certificates, licenses or permits is verifiable. 
 
 OpenAttestation documents are cryptographically secure and hence tamper-resistant. OpenAttestation also helps any user check who issued the document, and whether the document has been revoked so that it is no longer valid. These verifications are always performed at the point of viewing (before the document displays).
 
-#### How are OpenAttestation documents distributed?
+### What can I do using the OpenAttestation framework?
+
+You can perform the following actions using the OA framework:
+
+>**Note:** In the diagrams below, "source" means the user's local machine where the request comes from.
+
+* Create a wallet or DID
+
+![Create a wallet](/docs/docs-section/faq/create-wallet.png)
+
+* Deploy document store
+
+![Deploy document store](/docs/docs-section/faq/deploy-document-store.png)
+
+* Configure DNS
+
+![Configure DNS](/docs/docs-section/faq/config-dns.png)
```

Comment:
Configuring DNS is more of a prerequisite than an action, you cannot do this via the OA framework but it is required for the identify verification mechanism. How do you think we can best represent this? or we can remove this part entirely

Reply:
removed "Configure DNS"